mne/io/eeglab/tests/test_eeglab.py

```diff
@@ -747,5 +747,5 @@ def test_eeglab_drop_nan_annotations(tmp_path):
         ch_types=np.repeat("EEG", len(ch_names)),
     )
 
-    with pytest.raises(RuntimeWarning, match="1 .* have an onset that is NaN.*"):
+    with pytest.warns(RuntimeWarning, match="1 .* have an onset that is NaN.*"):
```

Comment:
Is this related?

Reply:
Related only because I made the `old` tests run more tests while I was doing some refactoring to make it easy for me to make the `old` env locally to debug a failure, and doing that exposed this issue. This always should have been a `pytest.warns`